dask/imperative.py

```diff
@@ -193,13 +196,11 @@ def compute(*args, **kwargs):
     >>> c = a + 3
     >>> compute(b, c)  # Compute both simultaneously
     (3, 4)
-    >>> compute(a, [b, c])
+    >>> compute(a, [b, c])  # Works for lists of Values
     (1, [3, 4])
     """
     args = [value(a) for a in args]
-    dsk = merge(*[arg.dask for arg in args])
-    keys = [arg.key for arg in args]
-    return tuple(get(dsk, keys, **kwargs))
+    return base.compute(*args, **kwargs)
```

Comment:
One question that came up while implementing this was whether this should be default behavior for `base.compute`. I can see the use of being able to say `compute([(val1, val2), (val3, val4)])`, and get results in the same structure as the input. I'm not sure if this same behavior is useful for the other collections, but a single common `compute` function across all of dask could be nice.